packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -169,7 +182,17 @@ export default {
     comments: '',
   }),
   methods: {
-    async submit() {
+    onSubmit() {
+      this.$refs.invisibleRecaptcha.execute();
+    },
+    onVerify(response) {
+      this.submit(response);
+    },
```

Comment:
No need to create a method to call another method -- make the `verify` argument passed to the component your `submit` method